account_ux/models/account_move.py

```diff
@@ -43,6 +44,13 @@ def _post(self, soft=True):
                     move_lines.ids
                 )
             ))
+        
+        user_lock_date = max(self.company_id.period_lock_date or date.min, self.company_id.fiscalyear_lock_date or date.min)
```

Comment:
def _post()

        for move in self:
            affects_tax_report = move._affect_tax_report()
            lock_dates = move._get_violated_lock_dates(move.date, affects_tax_report)
            if lock_dates:
                raise 
        return super()